<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="5" ignored="0" total="5" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="135515" started-at="2018-03-19T10:32:47Z" finished-at="2018-03-19T10:35:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="135515" started-at="2018-03-19T10:32:47Z" finished-at="2018-03-19T10:35:02Z">
      <class name="com.GoogleVsBing.GoogleVsBingCompareTest">
        <test-method status="PASS" signature="prepareClass()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="prepareClass" is-config="true" duration-ms="8498" started-at="2018-03-19T10:32:47Z" finished-at="2018-03-19T10:32:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareClass -->
        <test-method status="PASS" signature="prepareTest()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="prepareTest" is-config="true" duration-ms="21757" started-at="2018-03-19T10:32:55Z" finished-at="2018-03-19T10:33:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTest -->
        <test-method status="FAIL" signature="buscar(java.lang.String)[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="buscar" duration-ms="23743" started-at="2018-03-19T10:33:17Z" data-provider="5values" finished-at="2018-03-19T10:33:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[primero]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Primero - significado de primero diccionario] but found [Primero - Wikipedia, la enciclopedia libre]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Primero - significado de primero diccionario] but found [Primero - Wikipedia, la enciclopedia libre]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.GoogleVsBing.GoogleVsBingCompareTest.buscar(GoogleVsBingCompareTest.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buscar -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="cleanup" is-config="true" duration-ms="1394" started-at="2018-03-19T10:33:41Z" finished-at="2018-03-19T10:33:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method status="PASS" signature="prepareTest()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="prepareTest" is-config="true" duration-ms="4246" started-at="2018-03-19T10:33:42Z" finished-at="2018-03-19T10:33:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTest -->
        <test-method status="FAIL" signature="buscar(java.lang.String)[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="buscar" duration-ms="15104" started-at="2018-03-19T10:33:46Z" data-provider="5values" finished-at="2018-03-19T10:34:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Segundo]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Segundo a Segundo – Noticias de Chihuahua - …] but found [¿Qué es Segundo? - Su Definición, Concepto y Significado]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Segundo a Segundo – Noticias de Chihuahua - …] but found [¿Qué es Segundo? - Su Definición, Concepto y Significado]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.GoogleVsBing.GoogleVsBingCompareTest.buscar(GoogleVsBingCompareTest.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buscar -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="cleanup" is-config="true" duration-ms="1367" started-at="2018-03-19T10:34:02Z" finished-at="2018-03-19T10:34:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method status="PASS" signature="prepareTest()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="prepareTest" is-config="true" duration-ms="4311" started-at="2018-03-19T10:34:03Z" finished-at="2018-03-19T10:34:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTest -->
        <test-method status="FAIL" signature="buscar(java.lang.String)[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="buscar" duration-ms="13846" started-at="2018-03-19T10:34:07Z" data-provider="5values" finished-at="2018-03-19T10:34:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Tercero]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Tercero | Spanish to English Translation - SpanishDict] but found [Departamento de 5° 6° 7° Semestres/Tercero - UNAM]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Tercero | Spanish to English Translation - SpanishDict] but found [Departamento de 5° 6° 7° Semestres/Tercero - UNAM]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.GoogleVsBing.GoogleVsBingCompareTest.buscar(GoogleVsBingCompareTest.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buscar -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="cleanup" is-config="true" duration-ms="1181" started-at="2018-03-19T10:34:21Z" finished-at="2018-03-19T10:34:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method status="PASS" signature="prepareTest()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="prepareTest" is-config="true" duration-ms="3973" started-at="2018-03-19T10:34:22Z" finished-at="2018-03-19T10:34:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTest -->
        <test-method status="FAIL" signature="buscar(java.lang.String)[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="buscar" duration-ms="15336" started-at="2018-03-19T10:34:26Z" data-provider="5values" finished-at="2018-03-19T10:34:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Cuarto]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Cuatro - Noticias de última hora, deportes, programas y ...] but found [Cuarto - Wikipedia, la enciclopedia libre]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Cuatro - Noticias de última hora, deportes, programas y ...] but found [Cuarto - Wikipedia, la enciclopedia libre]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.GoogleVsBing.GoogleVsBingCompareTest.buscar(GoogleVsBingCompareTest.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buscar -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="cleanup" is-config="true" duration-ms="1132" started-at="2018-03-19T10:34:42Z" finished-at="2018-03-19T10:34:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method status="PASS" signature="prepareTest()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="prepareTest" is-config="true" duration-ms="4012" started-at="2018-03-19T10:34:43Z" finished-at="2018-03-19T10:34:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTest -->
        <test-method status="FAIL" signature="buscar(java.lang.String)[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="buscar" duration-ms="13908" started-at="2018-03-19T10:34:47Z" data-provider="5values" finished-at="2018-03-19T10:35:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Quinto]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Quinto - significado de quinto diccionario] but found [Quinto Bar]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Quinto - significado de quinto diccionario] but found [Quinto Bar]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.GoogleVsBing.GoogleVsBingCompareTest.buscar(GoogleVsBingCompareTest.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- buscar -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:com.GoogleVsBing.GoogleVsBingCompareTest@1fa268de]" name="cleanup" is-config="true" duration-ms="1599" started-at="2018-03-19T10:35:01Z" finished-at="2018-03-19T10:35:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
      </class> <!-- com.GoogleVsBing.GoogleVsBingCompareTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
